website/layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,10 @@
+{{ with .Site.Params.algolia_docsearch }}
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript"> docsearch({
+apiKey: '265a4e0c97efa2f55b1f8a95a20335d9',
+indexName: 'sdk-operatorframework',
+inputSelector: '.td-search-input',
+debug: false // Set debug to true if you want to inspect the dropdown
+});
+</script>
+{{ end }}
```

Comment:
```suggestion
{{ end }}

```

Reply:
add one empty caracter just to fix the CI because of the migration for the Travis.org